### DISCUSSION TITLE
HTML buttons (with tooltip?)

### DISCUSSION THREAD

**Discussion by dmi3kno** [quarto-dev/quarto-cli#4920] [Q&A] [answered]
I understand we can create clickable buttons by assigning a button class to some text with links. This is what I do:
```qmd
[Take the quiz!](quiz.qmd){.btn .btn-outline-primary .btn role="button"} 
```

What I would love to have is a tooltip activated on hover, like

```qmd
[Take the quiz!](quiz.qmd){.btn .btn-outline-primary .btn role="button" data-toggle="tooltip" title="Here's a tooltip"}
```

I understand that there should a [toggle JS function](https://getbootstrap.com/docs/4.0/components/tooltips/) included somewhere on the page. Can you guys JS wizards guide me whether it is possible to add a tooltip to my cool little button and where should I plug in the activating JS code? I understand that once we activate the tooltip, the placement tag should work as well.

**Comment by cderv** [comment, marked as answer]
Did you try already to use this ? 

Using this 

````markdown
---
title: "Untitled"
format: html
---

[Take the quiz!](quiz.qmd){.btn .btn-outline-primary .btn role="button" data-toggle="tooltip" title="Here's a tooltip"}

````

I got the tooltip on hover 
![image](https://user-images.githubusercontent.com/6791940/226939312-c598690b-9bd7-434a-99c0-894371d1ea92.png)

Not sure how / what activate this by default though. 

Are you using an other format ?

**Reply by dmi3kno** [reply]
oops I guess I solved my own question. It does not show in Rstudio preview, but visible in the browser. Thanks!

**Reply by drpawelo** [reply]
And unless you have custom css changing button styles, here are some examples of what is possible [https://www.w3schools.com/bootstrap4/bootstrap_buttons.asp](https://www.w3schools.com/bootstrap4/bootstrap_buttons.asp)

**Reply by mcanouil** [reply]
@drpawelo thanks. Please avoid resurrecting more than 3 years old thread (closed and answered) which triggers notification to all participant.

Also, note that you shared Bootstrap 4 examples while Quarto is using Bootstrap 5 since the start of the project.
